packages/components/src/Actions/ActionDropdown/ActionDropdown.snapshot.test.js

```diff
@@ -28,10 +28,10 @@ describe('ActionDropdown', () => {
 		};
 
 		// when
-		const wrapper = mount(<ActionDropdown {...props} />).find('ActionDropdown');
+		const wrapper = mount(<ActionDropdown {...props} />);
 
 		// then
-		expect(toJsonWithoutI18n(wrapper)).toMatchSnapshot();
+		expect(wrapper.html()).toMatchSnapshot();
```

Comment:
Turn every snapshot of this file from enzyme json interpretation to html.
Way easier to read with a simple component like that, and it removes 3/4 of the snapshot lines, removing lots of pollution.